unitTest/runMatRadTests.m

```diff
@@ -0,0 +1,24 @@
+%% This file runs the complete matRad test suite.
```

Comment:
This file does not follow the usual matrad naming convention even though it is an *.m file.

Reply:
true, same goes for `runtests.sh`?

Reply:
true but runtest.sh is not an *.m file so I would keep that. the matRad_ prefix is used so that we can distinguish matrad native functions from matlab native function within Matlab...